kolibri/plugins/facility/assets/src/views/users/sidePanels/UserCreate/index.vue

```diff
@@ -370,9 +370,9 @@
           data: {
             facility: activeFacilityId.value,
             username: username.value,
-            full_name: fullName.value,
+            full_name: fullName.value.trim(),
```

Comment:
@nucleogenesis I think we can keep  the current approach of trimming in the data object here?

Reply:
Yeah that works - it looks like reworking the `v-model` here to use `trim` might be a bit trickier since the value props are set directly w/ the `.sync`.

Might be worth trying adding `.trim` to the prop like:

```javascript
<FullNameTextbox
    ref="fullNameTextbox"
    // ... other props ...
    :value.sync.trim="fullName"    // note the .trim here
    // ... other props ...
  />
```

Although, that may only be available using `v-model`. 

In any case, no need to spend much time on that beyond seeing what you learn. Keeping the `.trim()` here is :+1: